scripts/download_checkpoints.bat

```diff
@@ -0,0 +1,2 @@
+mkdir ..\checkpoints

```

Comment:
..\checkpoints doesn't work if the .bat file is not run with the scripts/ directory as root (e.g. from the instruct-pix2pix/ directory like the instructions say)

Something like %~dp1..\checkpoints should probably be used.

```suggestion
mkdir %~dp1..\checkpoints

```

Reply:
Good call, I didnt think about that. Was just slapping together something to grab the checkpoint to get it off the ground.